introduccion-a-js-master/tareas/clase-6/task-2-class-6-scripts.js

```diff
@@ -0,0 +1,130 @@
+const $body = document.querySelector('body');
+const $buttonAddInput = document.querySelector('#add-input-button');
+const $buttonRemoveInput = document.querySelector('#remove-input-button');
+const $salaryFormContent = document.querySelector('#salary-form-content');
+const $salaryForm = document.querySelector('#salary-form');
+const $buttonCalculateSalary = document.querySelector('#calculate-salary-button');
+const $resultContent = document.querySelector('#result-values-salary');
+
+const $salaryFormTitle = document.createElement('h3');
+$salaryFormTitle.classList.add = "salary-form-title"
+$salaryFormTitle.innerText = "Enter the annual salary of each family member, only one salary per input";
+$salaryFormContent.insertBefore($salaryFormTitle, $salaryForm);
+
+$buttonAddInput.onclick = () => {
+    $salaryFormContent.style.display = "block";
+    $salaryFormTitle.style.display = "block";
+
+    const $salaryInputs = createLabelInput("Annual Salary: ", "salary-label", "salary-input");
+    $salaryForm.insertBefore($salaryInputs, $buttonCalculateSalary)
+
+    return false;
+}
+
+$buttonRemoveInput.onclick = () => {
+    const $allLabels = document.querySelectorAll('.salary-label');
```

Comment:
I think it's not necessary to add the ``all`` word in the variable, instead, you can declare ``const labels = ...`` that way you're referring to the ALL labels with ``'.salary-label'`` class.